starter-code/index.html

```diff
@@ -2,30 +2,50 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+  <div class="flexbox main">
+    <section>
+      <img src="./images/phones.png" alt="">
+    </section>
+    <section class="right-section">
+      <header class="flexbox">
+        <img src="./images/brand.png" alt="">
+        <button class="flexbox">
+          <img src="./images/home.png" alt="">
+          <p>Log in</p>
+        </button>
+      </header>
+      <div class="text">
+        <h2>Capture and Share<br/> the World's Moments</h2>
+        <p>Instagram is a <b>fast</b>, <b>beautiful</b> and <b>fun</b> way to share your life with friends and family.</p>
+        <p>Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
+        <p>Oh yeah, did we mention it's free?</p>
+      </div>
+      <div class="flexbox">
+        <div class="apps">
```

Comment:
This is actually a bit overkill in terms of containers you have. You could have simply had:
```html
<div class="apps">
    <img src="./images/badge-iphone.png" alt="">
    <img src="./images/badge-android.png" alt="">
</div>
```
Added the flexbox property to the "apps" class, and adding the margins to `.apps img {}`

We're talking optimization, but your solution works too :)